code/datums/jobs.dm

```diff
@@ -1508,7 +1508,6 @@ ABSTRACT_TYPE(/datum/job/civilian)
 
 /datum/job/special/random/director
 	name = "Regional Director"
-	receives_miranda = 1
```

Comment:
Probably not supposed to be in this PR? Sounded like it's supposed to be in #10891.